src/components/ServicesList/ServicesList.js

```diff
@@ -0,0 +1,19 @@
+import image1 from "./images/Group 46.png";
```

Comment:
Name it as the usage suggest so for example `image` is from file `groomer` so it should be `import groomerImage from '../../assets/groomerImage.png'`